Stellar-reviewable-request-asset.x

```diff
@@ -1,24 +1,30 @@
-
-
 %#include "xdr/Stellar-types.h"
 
 namespace stellar
 {
 
+//: AssetCreationRequest is used to create asset with provided parameters
 struct AssetCreationRequest {
-
-	AssetCode code;
-	AccountID preissuedAssetSigner;
-	uint64 maxIssuanceAmount;
-	uint64 initialPreissuedAmount;
+    //: Code of the asset to create
+    AssetCode code;
+    //: Public key of the signer that will perform pre issuance
+    AccountID preissuedAssetSigner;
+    //: Maximal amount to be issued
+    uint64 maxIssuanceAmount;
+    //: Amount to pre issue on asset creation
+    uint64 initialPreissuedAmount;
+    //: Bit mask of policies to create asset with
     uint32 policies;
+    //: Arbitrary stringified JSON object that can be used to attach data to be reviewed by the admin
     longstring creatorDetails; // details set by requester
+     //: Type of asset, selected arbitrarily. Can be used to restrict usage of asset
     uint64 type;
-
-	uint32 sequenceNumber;
+    //: Used to keep track of rejected requests update. On each rejected AssetCreationRequest update, sequenceNumber increases
```

Comment:
//: Used to keep track of rejected requests updates (`SequenceNumber` increases after each rejected AssetCreationRequest update)